### PR TITLE
Format logged objects to indicate their type

### PR DESCRIPTION
**Pull request by wbamberg**
Here's a simple attempt to format logged objects to indicate their type:

* wrap strings in quotes
* wrap arrays in square brackets

### CODE REVIEW

**Comment by wbamberg** [comment]
This is pretty basic actually. It doesn't understand objects in arrays:

```
console.log(["1", 1]
> [1,1]
```
Maybe we should add that?

File: js/editor-libs/console.js [hunk 1]
```diff
@@ -17,6 +17,22 @@
...
 
     /**
+    * Formats output to indicate its type:
+    * - quotes around strings
+    * - square brackets around arrays
+    * @param {any} input - The output to log.
+    */
+    function indicateType(input) {
+        if (typeof(input) === "string") {
+            return `"${input}"`;
```
Comment (schalkneethling) [inline js/editor-libs/console.js]: Unfortunately template literals are not supported in IE[1] and we still need to support at least v11 :(
[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: js/editor-libs/console.js [hunk 2]
```diff
@@ -17,6 +17,22 @@
     };
 
     /**
+    * Formats output to indicate its type:
+    * - quotes around strings
+    * - square brackets around arrays
+    * @param {any} input - The output to log.
+    */
```
Comment (schalkneethling) [inline js/editor-libs/console.js]: Nit: Indent lines 20-24 by one space

File: js/editor-libs/console.js [hunk 3]
```diff
@@ -17,6 +17,22 @@
...
+    function indicateType(input) {
```
Comment (schalkneethling) [inline js/editor-libs/console.js]: Nit: Wondering if `formatOutput` might be a more apt name?

File: js/editor-libs/console.js [hunk 4]
```diff
@@ -17,6 +17,22 @@
     };
 
     /**
+    * Formats output to indicate its type:
+    * - quotes around strings
+    * - square brackets around arrays
+    * @param {any} input - The output to log.
```
Comment (schalkneethling) [inline js/editor-libs/console.js]: Nit: Add `@returns Formatted output as a string`

File: js/editor-libs/console.js (hunk 1 above)
Comment (wbamberg) [reply js/editor-libs/console.js]: *sobs*

**Comment by wbamberg** [comment]
Thanks for the review @schalkneethling ! I've addressed the issues.

**Pull request merged**